src/app/components/mno-app-features/mno-app-features.html

```diff
@@ -0,0 +1,20 @@
+<div id="app-features">
+  <div class="btn-group">
+    <label for="frm-test-elm-100-1" class="btn btn-default">
```

Comment:
You don't need the a label here. The `<label>` tag defines a label for an `<input>` element.